fs/log/log.go

```diff
@@ -113,6 +113,8 @@ func InitLogging() {
 	}
 	log.SetFlags(flags)
 
+	fs.LogPrintPid = strings.Contains(flagsStr, ",pid,")
```

Comment:
This needs to go in the docs

https://github.com/rclone/rclone/blob/bb0c4ad2d8af8af1ff9abea54dbdf97eadc9018f/docs/content/docs.md#L1026

Reply:
done